### DISCUSSION TITLE
warning - Peering with (B) on vif 2 is one-way

### DISCUSSION THREAD

**Discussion by lucdig** [troglobit/mrouted#61] [Q&A]
Hi, with mrouted 4.5 installed on (A) and (B) I see, on (A) #mrouted -n:

warning - Peering with IP_(B) on vif 2 is one-way

On (B) , I don't see (A) in the Neighbor Table.

I don't understand why....

**Comment by troglobit** [comment]
This can happen if (link-local) multicast packets are not forwarded properly.
